src/adapters/supabase/helpers/user.ts

```diff
@@ -0,0 +1,58 @@
+import { Database } from "../types/database";
+import { SupabaseClient } from "@supabase/supabase-js";
+import { Super } from "./supabase";
+import { Context } from "../../../types/context";
+import { addCommentToIssue } from "../../../utils/issue";
+
+export class User extends Super {
+  constructor(supabase: SupabaseClient<Database>, context: Context) {
+    super(supabase, context);
+  }
+
+  async getWalletByUserId(userId: number, issueNumber: number) {
+    const { data, error } = await this.supabase.from("users").select("wallets(*)").eq("id", userId).single();
+    if ((error && !data) || !data.wallets?.address) {
+      console.error("No wallet address found", { userId, issueNumber }, true);
+      await addCommentToIssue(this.context, "```diff\n # Please set your wallet address with the /wallet command first and try again.\n```");
+      throw new Error("No wallet address found");
+    }
+
+    console.info("Successfully fetched wallet", { userId, address: data.wallets?.address });
```

Comment:
All the `console` logs should be coming from the `logger` itself I believe.

Reply:
And we should use `Logs.ts` or the simple `context` logger until the `ubiquity/logger` is refactored?

Or should it be fired up like:

```ts
context.adapters = createAdapters(supabase, context);
context.logger = new Logs(supabase, 0, "ERROR", context)
```

Reply:
@Keyrxng should be from the context.